include/RED4ext/Process.hpp

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <RED4ext/CString.hpp>
+#include <RED4ext/Common.hpp>
+#include <cstdint>
+
+namespace RED4ext
+{
+struct Process
+{
+    static constexpr uint32_t DefaultTimeout = 60000;
+
+    struct FixedWString
+    {
+        uint32_t length;
+        uint32_t maxLength;
+        const wchar_t* str;
+    };
+
+    enum class ExecutionFlags : std::uint8_t
+    {
+        // unsets CREATE_NO_WINDOW
+        ShouldCreateWindow = 0x1,
+        // sets CREATE_BREAKAWAY_FROM_JOB | CREATE_SUSPENDED in Process Creation Flags
+        BreakawayAndSuspend = 0x2,
+        // unsets bInheritHandles
+        NoInheritHandles = 0x4
+    };
+
+    enum class ReadFlags : std::uint8_t
+    {
+        // open & read regardless of other options
+        Unk1 = 0x1,
+        // opens file from name provided and writes output
+        WriteToFile = 0x2,
+        // writes output to CString
+        GetCString = 0x4
+    };
```

Comment:
I remembered that we decided to use struct flags, e.g. https://github.com/WopsS/RED4ext.SDK/blob/master/include/RED4ext/RTTITypes.hpp#L117-L132, so please convert these a as well to be consistent.